src/backend/api/wine.ts

```diff
@@ -21,6 +21,8 @@ export const installWineVersion = async (release: WineVersionInfo) =>
   ipcRenderer.invoke('installWineVersion', release)
 export const removeWineVersion = async (release: WineVersionInfo) =>
   ipcRenderer.invoke('removeWineVersion', release)
+export const refreshWineVersionInfo = async (fetch?: boolean) =>
+  ipcRenderer.invoke('refreshWineVersionInfo', fetch)
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const handleProgressOfWinetricks = (onProgress: any) =>
```

Comment:
```suggestion
export const handleProgressOfWinetricks = (onProgress: string[]) =>
```

Reply:
onProgress is a callback and the api method should return the removeListener call too. I tried using separate api methods for .on and .removeListener, and it was not unsubscribing in the tests that I ran.
```
export const handleProgressOfWinetricks = (
  onProgress: (e: Electron.IpcRendererEvent, messages: string[]) => void
) => {
  ipcRenderer.on('progressOfWinetricks', onProgress)
  return () => {
    ipcRenderer.removeListener('progressOfWinetricks', onProgress)
  }
}
```